up42/catalog.py

```diff
@@ -116,98 +110,116 @@ def place_order(
 
         Warning "Deprecated order parameters"
-            The use of the 'scene' and 'geometry' parameters for the data ordering is deprecated. Please use the new
+            The use of the 'scene' and 'geometry' parameters for
+            the data ordering is deprecated. Please use the new
             order_parameters parameter as described above.
 
          Warning:
-            When placing orders of items that are in archive or cold storage,
-            the order fulfillment can happen up to **24h after order placement**.
-            In such cases, please make sure to set an appropriate `report_time`.
-            You can also use `Order.track_status` on the returned object to track the status later.
+            When placing orders of items that are in
+            archive or cold storage,
+            the order fulfillment can happen up to
+            **24h after order placement**.
+            In such cases, please make sure to set
+            an appropriate `report_time`.
+            You can also use `Order.track_status` on the
+            returned object to track the status later.
 
         Returns:
             Order class object of the placed order.
         """
         if "scene" in kwargs or "geometry" in kwargs:
             # Deprecated, to be removed, use order_parameters.
             message = (
-                "The use of the 'scene' and 'geometry' parameters for the data ordering is deprecated. "
+                "The use of the 'scene' and 'geometry' parameters "
+                "for the data ordering is deprecated. "
                 "Please use the new 'order_parameters' parameter."
             )
             warnings.warn(message, DeprecationWarning, stacklevel=2)
         elif order_parameters is None:
             raise ValueError("Please provide the 'order_parameters' parameter!")
-
-        order = Order.place(self.auth, order_parameters)  # type: ignore
+        placed_order = order.Order.place(self.auth, order_parameters)  # type: ignore
         if track_status:
-            order.track_status(report_time)
-        return order
+            placed_order.track_status(report_time)
+        return placed_order
 
 
-class Catalog(CatalogBase, VizTools):
+class Catalog(CatalogBase, viztools.VizTools):
     """
-    The Catalog class enables access to the UP42 catalog functionality (data archive search & ordering).
+    The Catalog class enables access to the UP42 catalog
+    functionality (data archive search & ordering).
 
     Use catalog:
     ```python
     catalog = up42.initialize_catalog()
     ```
 
-    This class also inherits functions from the [CatalogBase](catalogbase-reference.md) class.
+    This class also inherits functions from the
+    [CatalogBase](catalogbase-reference.md) class.
     """
 
-    def __init__(self, auth: Auth):
+    def __init__(self, auth: up42_auth.Auth):
+        super().__init__(auth)
         self.auth = auth
```

Comment:
Not needed

Reply:
if I remove this, pylint complains with:
```W0231: __init__ method from base class 'CatalogBase' is not called (super-init-not-called)```

```up42/catalog.py:173:4: W0231: __init__ method from base class 'VizTools' is not called (super-init-not-called)```

Reply:
I meant self.auth = auth, it is handled in parent